src/main/java/calculator/domain/SplitTextNum.java

```diff
@@ -0,0 +1,26 @@
+package calculator.domain;
+
+import calculator.util.ValidUtil;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SplitTextNum {
+    private List<String> textNums;
+
+    public SplitTextNum(String[] textNums) {
+        List<String> listNums = trim(textNums);
+        ValidUtil.validateNumber(listNums);
+        this.textNums = listNums;
+    }
+
+    private List<String> trim(String[] textNums) {
+        return Arrays.stream(textNums).map(String::trim).collect(Collectors.toList());
```

Comment:
개인적으로 메서드 체이닝 사용시 가독성을 위해 한줄에 하나의 dot(.)만 사용하는게 좋다고 생각해요.
```suggestion
        return Arrays.stream(textNums)
                     .map(String::trim)
                     .collect(Collectors.toList());
```

Reply:
네 다음스탭부터는 메서드 체이닝 사용할때 가독성을 신경쓰도록 하겠습니다 
감사합니다